compose-files/extra-services/bitcoin/Dockerfile

```diff
@@ -1,12 +1,11 @@
 FROM debian:stretch-slim
-
 ARG BTC_VERSION=0.21.1
 ENV BTC_VERSION=${BTC_VERSION}
 WORKDIR /
 COPY ./make_binary_dist.sh /make_binary_dist.sh
 # COPY ./docker-entrypoint.sh /docker-entrypoint.sh
 # RUN chmod 755 /docker-entrypoint.sh /make_binary_dist.sh && mkdir -p /root/.bitcoin
-RUN chmod 755 /make_binary_dist.sh && mkdir -p /root/.bitcoin
+RUN chmod 755 /make_binary_dist.sh && mkdir -p /root/.bitcoin && mkdir -p /etc/bitcoin
```

Comment:
same applies to /root/.bitcoin that is also mounted at runtime. Both are redundant.
I added it to be consistent with the current code logic.
I would either leave it as it is or delete both. I'll delete them both on a new commit if you prefer (unless you tell me otherwise)

Reply:
makes sense - remove them both

Reply:
ok. removed `&& mkdir -p /etc/bitcoin`